tests/integration/test_cmorize_obs.py

```diff
@@ -0,0 +1,178 @@
+    }
+    config_file.write_text(yaml.safe_dump(cfg, encoding=None))
+    return str(config_file)
+
+
+def _create_sample_cube():
+    """Create a quick CMOR-compliant sample cube."""
+    coord_sys = iris.coord_systems.GeogCS(iris.fileformats.pp.EARTH_RADIUS)
+    cube_data = np.ones((2, 3, 2, 2))
+    cube_data[1, 1, 1, 1] = 22.
+    time = iris.coords.DimCoord([15, 45],
+                                standard_name='time',
+                                bounds=[[1., 30.], [30., 60.]],
+                                units=Unit('days since 0000-01-01',
+                                           calendar='gregorian'))
+    zcoord = iris.coords.DimCoord([0.5, 5., 50.],
+                                  var_name='depth',
+                                  standard_name='depth',
+                                  bounds=[[0., 2.5], [2.5, 25.], [25., 250.]],
+                                  units='m',
+                                  attributes={'positive': 'down'})
+    lons = iris.coords.DimCoord([1.5, 2.5],
+                                standard_name='longitude',
+                                bounds=[[1., 2.], [2., 3.]],
+                                units='K',
+                                coord_system=coord_sys)
+    lats = iris.coords.DimCoord([1.5, 2.5],
+                                standard_name='latitude',
+                                bounds=[[1., 2.], [2., 3.]],
+                                units='K',
+                                coord_system=coord_sys)
+    coords_spec = [(time, 0), (zcoord, 1), (lats, 2), (lons, 3)]
+    cube = iris.cube.Cube(cube_data, dim_coords_and_dims=coords_spec)
+    return cube
+
+
+def put_dummy_data(data_path):
+    """Create a small dummy netCDF file to be cmorized."""
+    gen_cube = _create_sample_cube()
+    t_path = os.path.join(data_path, "woa13_decav81B0_t00_01.nc")
+    # correct var names
+    gen_cube.var_name = "t_an"
+    iris.save(gen_cube, t_path)
+    s_path = os.path.join(data_path, "woa13_decav81B0_s00_01.nc")
+    gen_cube.var_name = "s_an"
+    iris.save(gen_cube, s_path)
+    # incorrect var names
+    o2_path = os.path.join(data_path, "woa13_all_o00_01.nc")
+    gen_cube.var_name = "o2"
+    iris.save(gen_cube, o2_path)
+    no3_path = os.path.join(data_path, "woa13_all_n00_01.nc")
+    gen_cube.var_name = "no3"
+    iris.save(gen_cube, no3_path)
+    po4_path = os.path.join(data_path, "woa13_all_p00_01.nc")
+    gen_cube.var_name = "po4"
+    iris.save(gen_cube, po4_path)
+    si_path = os.path.join(data_path, "woa13_all_i00_01.nc")
+    gen_cube.var_name = "si"
+    iris.save(gen_cube, si_path)
+
+
+def check_log_file(log_file, no_data=False):
+    """Check the cmorization log file."""
+    with open(log_file, 'r') as log:
+        if no_data:
+            cases = []
+            for line in log.readlines():
+                case = "Could not find raw data WOA" in line
+                cases.append(case)
+            assert True in cases
```

Comment:
Something like this
```suggestion
            msg = "Could not find raw data WOA"
            assert any(msg in line for line in log):
```
would be a bit nicer

Reply:
yip, cheers dude